swiftui-project/LoginView.swift

```diff
@@ -0,0 +1,142 @@
+//
```

Comment:
MVVM pattern is not implemented

Reply:
LoginViewModel implemented in folder View/Login